### PR TITLE
System page: various minor UI tweaks

### PR DESCRIPTION
**Pull request by leccelecce**
First PR, thank you for the excellent software!

Per discussion with Nick M on https://github.com/blakeblackshear/frigate/discussions/4978, this is a tiny PR to implement some minor UI changes to the System page. These were changes I could make without rebuilding core; there are more suggested changes on the discussion thread that would require backend changes as well which I will send a PR for separately once I have everything set up.

Changes:

-Add a CPU% column to the Detectors cards. I have tested this with CPU detectors and OpenVINO in CPU mode. I don't have the ability to run a GPU detector at present, so would appreciate a test by someone to ensure it doesn't break anything (it shouldn't).
-Added "ms" after inference speed value to make it clear what the number means
-Capitalized FPS, CPU acronyms
-Reordered per-camera table to follow data flow of ffmpeg => capture => detect

### CODE REVIEW

File: web/src/routes/System.jsx [hunk 1]
```diff
@@ -219,20 +221,27 @@ export default function System() {
                         <Tr>
                           <Th>Process</Th>
                           <Th>P-ID</Th>
-                          <Th>fps</Th>
-                          <Th>Cpu %</Th>
+                          <Th>FPS</Th>
+                          <Th>CPU %</Th>
```
Comment (NickM-27) [inline web/src/routes/System.jsx:225]: If you're going to capitalize CPU then you should also capitalize GPU

File: web/src/routes/System.jsx [hunk 2]
```diff
@@ -143,13 +143,15 @@ export default function System() {
...
+                        <Th>CPU %</Th>
                       </Tr>
                     </Thead>
                     <Tbody>
                       <Tr>
                         <Td>{detectors[detector]['pid']}</Td>
                         <Td>{detectors[detector]['detection_start']}</Td>
-                        <Td>{detectors[detector]['inference_speed']}</Td>
+                        <Td>{detectors[detector]['inference_speed']} ms</Td>
+                        <Td>{cpu_usages[detectors[detector]['pid']]?.['cpu'] || '- '}%</Td>
```
Comment (NickM-27) [inline web/src/routes/System.jsx:154]:
```suggestion
                        <Td>{cpu_usages[detectors[detector]['pid']]?.['cpu'] || '-'}%</Td>
```

**Review by NickM-27 — CHANGES_REQUESTED** [review]
Looks good. Tried it and seems fine with corals as well, I think it should make sense to just leave this % in for all detectors as it may be useful

Screen Shot 2023-01-09 at 17 03 50 PM

File: web/src/routes/System.jsx (hunk 1 above)
Comment (leccelecce) [reply web/src/routes/System.jsx:225]: Ah yes! I don't have a GPU in my current test system so can't see that...will ensure I retest on another box in future.

File: web/src/routes/System.jsx (hunk 2 above)
Comment (leccelecce) [reply web/src/routes/System.jsx:154]: I actually copied this command (including that trailing space) from the Camera table below. Do you want it removed there to? I assumed it was like that for a reason

Comment (NickM-27) [reply web/src/routes/System.jsx:154]: Oops yeah I read that wrong.

**Review by NickM-27 — APPROVED** [review]

**Comment by blakeblackshear** [comment]
Do we still think the detection start time is useful here? I'm not sure it is.

**Comment by NickM-27** [comment]
I think like in the discussion linked above it might be more useful if it shows a state like `idle` or "started x milliseconds ago" but yeah not sure it's needed for most cases

**Comment by leccelecce** [comment]
I just experimented with this a bit and I'm struggling to get an obviously useful statistic out of the last detection start time if I calculate _now() - start_ it in the front end, I'm guessing because the stats are updating periodically on the server and so the last start time might be ~ 10 seconds out of date by the time the front end refreshes.

In the short term, I have just changed it in the UI to be 'running' or 'idle', sample below. 

image

As I'm planning a separate PR with some more tweaks that require server side changes, I will look at revisiting this to provide a "started X ms ago"/"idle" instead of "running"/"idle" if that's what you would prefer.

**Comment by leccelecce** [comment]
One interesting thing the last change highlights is that it's often possible for no detection to be running, but CPU to still be high. I'm guessing that's because the `top` command for CPU util is not by nature exactly aligned to the detection code, or possibly that the CPU has a bit more work to do after the detection timer finishes. For that reason it might be more useful under Detection Status to have a "detections last 1s" or similar?

**Comment by blakeblackshear** [comment]
I was suggesting just removing it. It's going to flip back and forth between running and idle so frequently, I am not sure it means much. Frigate is already watching and restarts the detection process with log messages if needed.

**Comment by leccelecce** [comment]
Removed!

**Comment by NickM-27** [comment]
Looks good to me 👍

**Pull request merged**